packages/contract-state-stream/__tests__/node_env.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @jest-environment node
+ */
+const Ganache = require("ganache-core");
+const Web3 = require("web3");
+const createNewBlock$ = require("@drizzle-utils/new-block-stream");
+const { take, finalize, tap, toArray } = require("rxjs/operators");
+
+const compile = require("./utils/compile");
+const createContractState$ = require("../index");
+
+jest.setTimeout(20000);
+
+describe("contract-state-stream tests in node environment", () => {
+  let provider;
+  let web3;
+  let accounts;
+  let contractInstance;
+  let artifact;
+
+  beforeAll(async () => {
+    // 1. Compile contract artifact
+    const { SimpleStorage } = await compile("SimpleStorage.sol");
+
+    // 2. Spawn Ganache test blockchain
+    provider = Ganache.provider({ seed: "drizzle-utils" });
+    web3 = new Web3(provider);
+    accounts = await web3.eth.getAccounts();
+
+    // 3. Create initial contract instance
+    const instance = new web3.eth.Contract(SimpleStorage.abi);
+
+    // 4. Deploy contract and get new deployed Instance
+    const deployedInstance = await instance
+      .deploy({ data: SimpleStorage.bytecode })
+      .send({ from: accounts[0], gas: 150000 });
+
+    // Note: deployed address located at `deployedInstance._address`
+
+    contractInstance = deployedInstance;
+
+    artifact = {
+      ...SimpleStorage,
+      // truffle-decoder needs this in artifact
+      networks: {
+        "4447": {
+          address: deployedInstance._address,
+        },
+      },
+    };
+  });
+
+  afterAll(async () => {
+    provider.close();
+  });
+
+  test("ensure test suite setup works", async () => {
+    // Get old value, set a new value, check to see if it was set correctly
+    const oldVal = await contractInstance.methods.get().call();
+    await contractInstance.methods.set(5).send({ from: accounts[0] });
+    const newVal = await contractInstance.methods.get().call();
+
+    expect(oldVal).toBe("0");
+    expect(newVal).toBe("5");
+  });
+
+  test("createContractState$ function exists", () => {
+    expect(createContractState$).toBeDefined();
+  });
+
+  test("createContractState$ throws errors when required options fields not found", async () => {
+    expect(() => createContractState$()).toThrow();
+
+    expect(() =>
```

Comment:
I think Prettier can put this into 2 lines instead of 6. Example [here](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHOYYApcCUABALwB8RYATnAIbwDC0MVtOAyjPXACS7BFaVGAEsAZmxgAaIhioQAbiIAmcKkQC+BAgDoYEACoALeQHdcUOKaIBRKvKq4AOiGNwiEDKOgBnDwCMAK2wYIlMRGCMiS1MAIQAbCDAAax4iET8aAEcAVxEaZRdtAG4QKRBPbygfZFAhMwAFIQQalFoFCBUykH9WFLgYdgw2ESgAc2QWHLhyoxgAW3iAdSMIuB9hsDh2FoiRJRgAT2RwHxry0Z81GAbWMfnaZAl4q-LAnzRYvuSB9lp5uAAGVGcCetBeMxA7zQ7FGY3icAAijkIPAwRDysMqFcqCd-LR-IdElBunJRjAlipIsgABwABkx8iuS1YGBOcnWagUoPK2TyNFutHujyQz1eICu8xEkyo03KPjhCORqNBovB4q4-kpympSAATOUWLQRPE4Ux5g8TlBoDyQDkrgYCa0xXANBogA).

Reply:
What's the actual prettier config field that controls this?

Reply:
There is none. Prettier takes cues from your line breaks. So if you had line breaks already, then it'll assume that you don't want it to be one line. However, if you try to put the object literal into one line, then it'll try not to break that portion of it.

Reply:
@honestbonsai See updated [example](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHOYYApcCUABALwB8RYATnAIbwDC0MVtOAyjPXACS7BFaVGAEsAZmxgAaIhioQAbiIAmcKkQC+BAgDoYEACoALeQHdcUOKaIBRKvKq4AOiGNwiEDKOgBnDwCMAK2wYIlMRGCMiS1MAIQAbCDAAax4iET8aAEcAVxEaZRdtAG4nKDL0LBx8YnJKGm4mWFYOLng+YDKiQWFxSRk5RRU1TW09QxMIcxjbewhHFzcPLxFfAOCcMIiomISk1PTMuFz8uEKQEpApEE9vKB9kUCEzAAUhBAeUWgUIFSuQfwtZJwGDsDBsERQADmyBYOTg1yMMAAtvEAOpGCJwHzgsBwdgfCIiJQwACeyHAPge10hPjUMBerChyNoyAk8Tp10CPjQsSBIPYtGRcAAMpC4GzaByESBuWh2JCofE4ABFHIQeCS6XXcFUOlUCn+Wj+UmJKD-OSQmBolSRZAADgADDr5HS0awMBS5Ni1AoJddsnkaIzaMzWUh2ZyQHTkSJYVR4dcfIrlWqNRKI1Ko1x-DblHakAAma4sWgieKKpjIlkUqDQf0gHJ0gzGz6RuAaDRAA).

Reply:
It's probably my local Prettier adjusting it to some default breaks

Reply:
Yea fair points, I think the disagreement comes down on the readability of this. I don't think it's unreadable. I did skim over it before submission and it was acceptable to me. I'd like it to remove as much of these discussions as possible. I will correct this though.